deptool/src/main/java/io/jmix/dependency/cli/command/ExportCommand.java

```diff
@@ -66,6 +86,22 @@ public FileVisitResult visitFile(Path file, BasicFileAttributes attrs) throws IO
                     return FileVisitResult.CONTINUE;
                 }
             });
+
+            if (!exportedDependencies.isEmpty()) {
+                try {
+                    Files.deleteIfExists(Paths.get(reportFile));
+                } catch (IOException e) {
+                    throw new RuntimeException("Error while creating the report file", e);
```

Comment:
Probably, it's an error while deleting an existing report file, not creating the one?